src/components/Photography/ImageZoomGridElement.tsx

```diff
@@ -1,72 +1,46 @@
 import Img from 'gatsby-image'
-import { ExifData } from 'gatsby-source-s3-image'
 import _ from 'lodash'
 import React, { MouseEventHandler } from 'react'
+import { Box } from 'rebass/styled-components'
 import styled from 'styled-components'
 
 import ExifOverlay from 'components/Photography/ExifOverlay'
-import GatsbyImage from 'types/GatsbyImage'
-import { mapLensModelExif } from 'utils/exif'
+import S3ImageAsset from 'types/S3ImageAsset'
 import useHover from 'utils/use-hover'
 
-const StyledImageZoomGridElement = styled.div`
+const StyledImageZoomGridElement = styled(Box)`
   border-radius: 2px;
   cursor: pointer;
-  display: flex;
-  flex-direction: column;
-  grid-column-start: auto;
   overflow: hidden;
-  position: relative;
-  transition: all 0.4s;
-  justify-content: flex-end;
+  transition: filter 0.5s;
+
   // Throw in some drop shadow to make it pretty (extra on hover) — and just a
-  // smidge of border radius, too:
-  filter: drop-shadow(0 0 2px rgba(0, 0, 0, 0.75));
+  //a smidge of border radius, too:
+  filter: drop-shadow(0 0 2px rgba(0, 0, 0, 0.5));
   &:hover {
     filter: drop-shadow(0 0 6px rgba(0, 0, 0, 0.9));
   }
 `
 
 interface Props {
-  image: {
-    id: any
-    childImageSharp: GatsbyImage
-    EXIF: ExifData
-  }
+  style?: any
+  image: S3ImageAsset
   onClick: MouseEventHandler<HTMLDivElement>
 }
 
 const ImageZoomGridElement = (props: Props) => {
   const { image } = props
-
   const [isActive, setIsActive] = useHover()
+  const thumbnailImage = _.get(image, 'childImageSharp.thumbnailSizes')
 
-  const thumbnailSizes: GatsbyImage = _.get(image, 'childImageSharp.sizes')
-
-  if (_.isEmpty(thumbnailSizes)) {
+  if (_.isEmpty(thumbnailImage)) {
     return null
   }
 
-  const { aspectRatio } = thumbnailSizes
-
-  const lensModel = mapLensModelExif(_.get(image, 'EXIF.LensModel'))
-  const { FNumber, ISO, FocalLength, ShutterSpeedFraction } = image.EXIF
-
   return (
-    <StyledImageZoomGridElement
-      {...props}
-      {...setIsActive}
-      style={aspectRatio > 1 ? { gridRow: 'span 1' } : { gridRow: 'span 2' }}
-    >
-      <Img fluid={thumbnailSizes} />
-      <ExifOverlay isActive={isActive as boolean}>
-        {FocalLength ? `${FocalLength}mm, ` : null}
-        {ShutterSpeedFraction ? `${ShutterSpeedFraction}s, ` : null}
-        {FNumber ? `ƒ${FNumber}, ` : null}
-        {ISO ? `ISO ${ISO}` : null}
-        <br />
-        {lensModel}
-      </ExifOverlay>
+    <StyledImageZoomGridElement {...props} {...setIsActive} margin={2}>
+      <Img imgStyle={{ objectFit: 'contain' }} fluid={thumbnailImage} />
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 12 spaces but found 6. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)